src/content/1.7/code/agda/snippet08.agda

```diff
@@ -0,0 +1 @@
+fmap-id-lemma : ∀ x → (fmap id) x ≡ id x
```

Comment:
Direct translation from Haskell `fmap id = id` would be:
```suggestion
fmap-id-lemma : (fmap id) ≡ id
```

Reply:
But then proves require extensionality :)

Reply:
Correct.  I wasn't sure what to do here.  Should we stick as close to Haskell as possible, even if it means we can't prove something? 🤔